MVCDemoApp/Models/SampleData.cs

```diff
@@ -0,0 +1,365 @@
+using System;
```

Comment:
Remove this file. Please check the ReadMe file and add Model accordingly.

Reply:
Managed it